.github/workflows/linters.yml

```diff
@@ -8,13 +8,16 @@ jobs:
     name: Lint
     runs-on: ubuntu-latest
     steps:
+    - uses: actions/setup-go@v3
+      with:
+        go-version: "1.23"
     - name: Checkout code
       uses: actions/checkout@v2
-    - name: Install golangci-lint 
+    - name: Install golangci-lint
       run: |
         go version
-        curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.23.8
+        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v2.1.6
     - name: Run required linters in .golangci.yml plus hard-coded ones here
-      run: $(go env GOPATH)/bin/golangci-lint run --timeout=3m
+      run: $(go env GOPATH)/bin/golangci-lint run --timeout=3m ./
     - name: Run optional linters (not required to pass)
-      run: $(go env GOPATH)/bin/golangci-lint run --timeout=3m --issues-exit-code=0 -E dupl -E gocritic -E gosimple -E lll -E prealloc
+      run: $(go env GOPATH)/bin/golangci-lint run --timeout=3m --issues-exit-code=0 -E dupl -E gocritic -E lll -E prealloc ./
```

Comment:
I believe this change should be reflected in the top-level Makefile by amending the `GOLINT_ARGS` variable.